pkgs/build-support/go/module.nix

```diff
@@ -297,6 +297,7 @@ let
     } // meta;
   });
 in
+lib.warnIf (args' ? vendorSha256) "`vendorSha256` is deprecated. Use `vendorHash` instead"
```

Comment:
Are we not missing a third argument to `warnIf`, as its docstring says? After I've updated `nixpkgs` in the flake definition where I keep my NixOS configuration, I get an error whenever I run `nixos-rebuild switch --flake . --show-trace`:

[nixos-rebuild-switch-flake-dot-show-trace.txt](https://github.com/NixOS/nixpkgs/files/14115855/nixos-rebuild-switch-flake-dot-show-trace.txt)